src/index.html

```diff
@@ -1,27 +1,52 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1.0"
-    />
-    <title>Catalog</title>
-    <link
-      rel="preconnect"
-      href="https://fonts.gstatic.com"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
-      rel="stylesheet"
-    />
-    <link
-      rel="stylesheet"
-      href="styles/index.scss"
-    />
-  </head>
-
-  <body>
-    <h1>Catalog</h1>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Catalog</title>
+  <link rel="preconnect" href="https://fonts.gstatic.com" />
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet" />
+  <link rel="stylesheet" href="styles/index.scss" />
+</head>
+
+<body>
+  <div class="imac-container">
+
+    <div class="cards">
+      <div class="imac"></div>
+    </div>
+  </div>
+
+  <h1><b>Моноблок Apple A1419 iMac 27" Retina 5K (MNED2UA/A)</b></h1>
+
+  <h2>Код товара: 195434</h2>
+
+
+  <div class="stars">
+    <div class="star"></div>
+    <div class="star"></div>
+    <div class="star"></div>
+    <div class="star"></div>
+    <div class="star"></div>
+    <h2><a href="">Отзывов:5</a></h2>
+  </div>
+
+  <div class="price">
+    <h1>Цена:</h1>
+    <h2><b>69 999 грн</b></h2>
+  </div>
+
+  <div class="buy">
+    <h1><a href="">Купить</a></h1>
```

Comment:
no need for an extra **`div`** container, the **`a`** tag itself is a container for its' text